system/config/templates/features/enroll_app/hbx_admin_config.yml

```diff
@@ -13,3 +13,5 @@ registry:
           is_enabled: true
         - key: :inbox_tab
```

Comment:
Why did you add a new feature? Shouldn't you just be using `EnrollRegistry.feature_enabled?(:aca_individual_market)`?